profile_library/signify/LWA029/model.json

```
@@ -0,0 +1,18 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "HOPI HP-9800",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 1,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.2.0:docker"
+    },
+    "name": "Hue White Bulb A19 E26 1100lm with Bluetooth",
+    "standby_power": 0.2,
+    "calculation_strategy": "lut",
+    "aliases": [
+        "LWA029",
+        "9290024691A"
+    ],
```

```suggestion
    "aliases": [
        "9290024691A"
    ],
```

This alias can be removed as the directory name is already that model id. Powercalc looks at the directory first.